### DISCUSSION TITLE
사용자 유치 계획 아이디어 취합

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#476] [아이디어]
사용자 유치 계획을 어떤 식으로 구성할지 아이디어 남겨주세요~

**Comment by nlom0218** [comment]
공시생, 수험생들이 있는 카페에 홍보를 한다.

**Comment by aak2075** [comment]
https://study-hyukyong.tistory.com/entry/%EC%B4%88%EA%B8%B0-%ED%8A%B8%EB%9E%98%ED%94%BD-%EC%9C%A0%EC%A0%80%ED%99%95%EB%B3%B4-%EB%B3%BC%EB%A7%81%ED%95%80-gross-%EB%A0%88%EB%B2%84-criticalmass

이런 전략들을 사용해보면 좋을것 같아요

**Comment by MoonJeWoong** [comment]
- 개발자 취준 잡담방 등 오카방 홍보
  - 우리가 직접 오카방을 만들어서 운영해봐도 괜찮을 듯!
  - 하루스터디 오카방을 만들어서 스터디 함께 진행할 사람 모집하는 공간으로 운영해봐도 좋을 것 같습니다!
- 개발 학습 커뮤니티 등에 홍보

**Comment by woosung1223** [comment]
### 카드뉴스 제작

하루스터디를 간략하게 소개할 수 있는 카드뉴스(3~4장 정도?)를 제작해서,
대학생들이 자주 사용하는 에브리타임이나 인스타그램 같은 곳에서 홍보하면 어떨까?

**Comment by woo-jk** [comment]
### 벨로그 트렌딩 이용
벨로그에 글을 올리고 팀원들이 단체로 좋아요를 누른다. 좋아요 7개 정도면 주간 트렌딩에 꽤 상위권으로 올라간다.

**Comment by jaehee329** [comment]
출신 학교 에브리타임 홍보 -> 20대 사용자들 유치에 유리

**Comment by yeopto** [comment]
주변 개발자/지망생에게 홍보
